code/workspace/flipping-formats-and-cleanup/example_parameter.json

```diff
@@ -0,0 +1,44 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Description": "AWS CloudFormation workshop - flipping formats and cleaning up - example parameter (uksb-1q9p31idr)",
+    "Parameters": {
+        "ParameterName": {
+            "Description": "Name you wish to specify for your SSM Parameter.",
+            "Type": "String",
+            "MinLength": 1,
+            "MaxLength": 2048
```

Comment:
Not that it matters much for this example but as per <https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-paramstore-su-create.html#sysman-parameter-name-constraints>:
> The maximum length for a parameter name, including the full content of the parameter Amazon Resource Name (ARN), is 1011 characters.

Reply:
Thank you, @stilvoid .  I think it is best to leave `MaxLength` to `2048` for consistency with following pages that I used as a reference:
- https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ssm-parameter.html#cfn-ssm-parameter-name
- https://docs.aws.amazon.com/systems-manager/latest/APIReference/API_PutParameter.html#systemsmanager-PutParameter-request-Name